src/js/EditController.js

```diff
@@ -26,13 +26,18 @@ goog.require('ga_mvt_service');
     $scope.options = angular.extend(options, $scope.options || {});
     $scope.globals = $scope.globals || {};
 
-    $scope.$on('gaToggleEdit', function(evt, layer) {
-      var toggle = !!(!$scope.globals.isEditActive ||
-        !$scope.layer ||
-        (layer && layer.bodId !== $scope.layer.bodId));
+    $scope.$on('gaToggleEdit', function(evt, layer, toggle) {
+      if (toggle === undefined) {
+        toggle = !!(!$scope.globals.isEditActive ||
+          !$scope.layer ||
+          (layer && layer.bodId !== $scope.layer.bodId));
+      }
       $scope.layer = layer;
       $scope.globals.isEditActive = toggle;
       $scope.globals.pulldownShown = toggle;
+      if (toggle) {
+        gaMvt.reload($scope.layer);
```

Comment:
we don't want to reload... We want to apply the style attached to layer

Reply:
Ok here is what I had in mind: https://github.com/geoadmin/mf-geoadmin3/pull/4580/commits/08548434a050d5026bfc844057e397f0d04ea475